### DISCUSSION TITLE
Duda sobre pasar informacion entre componentes en vistas distintas

### DISCUSSION THREAD

**Discussion by MauricioNovelo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#565] [Q&A]
No e podido pasar la informacion entre los componentes, en la misma vista si se ven los resultados que se ponen en el input, pero cuando pongo el componente en otra vista ya no muestra la informacion.

**Comment by ismagician** [comment]
Si usas Vuex en el store se almacena información, en el último live de Vuejs el MC lo explica
https://web.microsoftstream.com/video/6fa3c8b3-a840-4cf2-998a-04007f8bc0ab

**Comment by oscardanielM** [comment]
Así como dice @ismagician je aunque igual puede ser la forma en como mandas a llamar las variables del store

**Comment by HugoHerreraR** [comment]
Puedes usar el local localstorage
